viz_scripts/energy_calculations.ipynb

```diff
@@ -106,7 +106,7 @@
    },
    "outputs": [],
    "source": [
-    "expanded_ct, file_suffix, quality_text, debug_df = scaffolding.load_viz_notebook_data(year,\n",
+    "expanded_ct, file_suffix, quality_text, debug_df = await scaffolding.load_viz_notebook_data(year,\n",
```

Comment:
don't we need the values_to_translations here? we should use the display modes in the energy and emission impact plots.

Reply:
We do not need the values_to_translations() here. It's being used only with the Stacked Bar Charts. We are using display modes in the energy and emission impact plots.

Reply:
Wait, so we have `mode_confirm`, `mode_confirm_w_other` and `Mode_confirm` all at the same time? When we remove the csvs I think we should probably get rid of `Mode_confirm` and just use `values_to_translations` to prevent confusion.